R/table.R

```diff
@@ -3,21 +3,30 @@ setClass("Table", slots = list(name = "character"))
 
 #' Refer to a table nested in a hierarchy (e.g. within a schema)
 #'
+#' Objects of class `Table` have a single slot `name`, which is a named
```

Comment:
Calling this a table seems slightly confusing since it's the name of a table, not a table itself. Maybe TableRef?

Reply:
And doesn't every other object start with `db`?

Reply:
`Table()` doesn't need the connection, it's just a dumb object like `SQL()`. We could use `Identifier()` too.

Reply:
I would suggest `Id()`

Reply:
`Id()` looks very prone to collisions with other packages...

Reply:
`SqlId`?

Reply:
There are no packages on CRAN which export an `Id` function

https://github.com/search?q=user%3Acran+%22export+Id%22+file%3ANAMESPACE&type=Code&utf8=%E2%9C%93

Reply:
This search doesn't find packages that export `crossing`: https://github.com/search?utf8=%E2%9C%93&q=user%3Acran+%22export+crossing%22+file%3ANAMESPACE&type=Code . I remember a recent discussion on Twitter about searching for packages that export a function?

`SqlId()` looks inconsistent to `SQL()`, and `SQLId()` looks ... interesting.

Reply:
Why doesn't it find "nest"? https://github.com/search?utf8=%E2%9C%93&q=user%3Acran+filename%3ANAMESPACE+%22export+nest%22&type=

Reply:
I am not sure, but it seems like GitHub limits the length of time a query can run. Anyway I ran the search on my Local CRAN mirror, there are no CRAN packages that export `Id`.

You can call it `Identifier` if you like it better.

Reply:
Confirmed (from a scraping of my local CRAN mirror) that no package exports an `Id` class either. Renaming.